spec/yaml/swiftnav/sbp/navigation.yaml

```diff
@@ -213,7 +213,7 @@ definitions:
             type: u16
             units: mm
             desc: |
-              Position accuracy estimate. 
```

Comment:
yeah this was an unintentional change (my edit strips trailing whitespace on save) but sure ill add the units